Docs/developer_guide/script_repository.md

```diff
@@ -8,6 +8,15 @@ To run a Python code snippet automatically at each application startup, add it t
 
 :::
 
+:::{note}
+
+**Fully worked examples**:
```

Comment:
What do you mean by "fully worked"? Complete examples?

Reply:
Well I meant the opposite of "snippet", so it could be changed to "Use in context" maybe just "More code for reference"

Reply:
"More code for reference" sounds good to me.

Reply:
Updated the name